docs/algorithms/Divide and Conquer Algorithm/Strassen'sAlgorithm.md

```diff
@@ -0,0 +1,165 @@
+---
```

Comment:
first you need rename your file `Strassen'sAlgorithm.md` to `strassen-algorithm.md` for best practices